### PR TITLE
Add EL9 and EL8 KVM guest image links

### PR DESCRIPTION
**Pull request by mjivraja**
There is a need to remove KVM guest image links of EL6 and EL7 for project versions above 3.3 where these OS versions are no longer supported. Therefore, we have added the links to EL9 and EL8 by replacing the old links.

https://bugzilla.redhat.com/show_bug.cgi?id=2133957

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [X] Foreman 3.5/Katello 4.7
* [X] Foreman 3.4/Katello 4.6
* [X] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Review by ekohl — COMMENTED** [review]
The last line in the file has xrefs to RHEL 6 & RHEL 7 image creation. Should those be updated too to keep in sync?

So from the preview: https://theforeman-foreman-documentation-preview-pr-1777.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Creating_a_RHEL7_Image_provisioning & https://theforeman-foreman-documentation-preview-pr-1777.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Creating_a_RHEL6_Image_provisioning

**Comment by adamlazik1** [comment]
> The last line in the file has xrefs to RHEL 6 & RHEL 7 image creation. Should those be updated too to keep in sync?

Agreed.

**Comment by mjivraja** [comment]
> The last line in the file has xrefs to RHEL 6 & RHEL 7 image creation. Should those be updated too to keep in sync?
> 
> So from the preview: https://theforeman-foreman-documentation-preview-pr-1777.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Creating_a_RHEL7_Image_provisioning & https://theforeman-foreman-documentation-preview-pr-1777.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Creating_a_RHEL6_Image_provisioning

Agreed. Pinged to the relevant team for getting procedures for EL9 and EL8. :+1:

**Comment by mjivraja** [comment]
> The last line in the file has xrefs to RHEL 6 & RHEL 7 image creation. Should those be updated too to keep in sync?
> 
> So from the preview: https://theforeman-foreman-documentation-preview-pr-1777.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Creating_a_RHEL7_Image_provisioning & https://theforeman-foreman-documentation-preview-pr-1777.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Creating_a_RHEL6_Image_provisioning

@ekohl Will this [link](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_a_customized_rhel_system_image/index) work? In this case, we may need to redirect instead of `xref`

**Comment by ekohl** [comment]
I think so, but then I would drop the entire appendix C1 and redirect users to that guide if they want to build RHEL images. Then C2 and C3 also become redundant.

@maximiliankolb any thoughts?

**Comment by mjivraja** [comment]
> I think so, but then I would drop the entire appendix C1 and redirect users to that guide if they want to build RHEL images. Then C2 and C3 also become redundant.
> 
> @maximiliankolb any thoughts?

Agree. C1, C2 and C3 can be dropped if we are redirecting. I have asked again to have a dedicated link for EL7 (same as EL9 and EL8). This will ensure uniformity in doc.

**Comment by maximiliankolb** [comment]
> Here we got the link to [EL7](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/image_builder_guide/index).
> 
> @ekohl @maximiliankolb We can drop C1, C2, and C3 to redirect here as we have got the links of all three EL versions.

@mjivraja I don't follow. Your change looks good to me; do you want to add additional changes?

**Comment by mjivraja** [comment]
> @mjivraja I don't follow. Your change looks good to me; do you want to add additional changes?

I added the changes with valid URLs. If this looks good to you, I will make the same changes in the corresponding PR #1778  :+1:

File: guides/common/modules/con_building_cloud_images.adoc [hunk 1]
```diff
@@ -5,8 +5,9 @@ Use this section to build and register images to {ProjectName}.
 
 You can use a preconfigured {RHEL} KVM guest QCOW2 image:
 
-* https://access.redhat.com/downloads/content/69/ver=/rhel---7/[Latest RHEL 7 KVM Guest Image]
-* https://access.redhat.com/downloads/content/69/ver=/rhel---6/6.10/x86_64/product-software[Latest RHEL 6 KVM Guest Image]
+* https://access.redhat.com/downloads/content/479/ver=/rhel---9/9.0/x86_64/product-software[Latest RHEL 9 KVM Guest Image]
+* https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.0/x86_64/product-software[Latest RHEL 8 KVM Guest Image]
+
```
Comment (maximiliankolb) [inline guides/common/modules/con_building_cloud_images.adoc:10]:
```suggestion
```

File: guides/common/modules/con_building_cloud_images.adoc [hunk 2]
```diff
@@ -20,4 +21,4 @@ For the KVM guest images:
 The `root` password is locked in `/etc/shadow` by placing `!!` in the second field.
 ====
 
-If you want to create custom {RHEL} images, see xref:Creating_a_RHEL7_Image_{context}[Creating a {RHEL} 7 Image] and xref:Creating_a_RHEL7_Image_{context}[Creating a {RHEL} 6 Image].
+If you want to create custom {RHEL} images, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/index[Composing a customized {RHEL} 9 Image] or https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_a_customized_rhel_system_image/index[Composing a customized {RHEL} 8 Image].
```
Comment (maximiliankolb) [inline guides/common/modules/con_building_cloud_images.adoc:24]:
```suggestion
ifdef::satellite[]
If you want to create custom {RHEL} images, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/index[Composing a customized {RHEL} 9 Image] or https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_a_customized_rhel_system_image/index[Composing a customized {RHEL} 8 Image].
endif::[]
```

**Review by maximiliankolb — COMMENTED** [review]
Two small suggestions, rest LGTM!

File: guides/common/modules/con_building_cloud_images.adoc (hunk 2 above)
Comment (mjivraja) [reply guides/common/modules/con_building_cloud_images.adoc:24]: Addressed :+1:

File: guides/common/modules/con_building_cloud_images.adoc (hunk 1 above)
Comment (mjivraja) [reply guides/common/modules/con_building_cloud_images.adoc:10]: Addressed :+1:

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Unfortunately, I've merged a "!fixup" commit into master. I was hoping GH would do a "real" rebase and automatically ~squash~ fixup the commit correctly. 🤷 

Merged to master and cherry-picked: ✅ 
   283193c7d..65c87a7b8  3.5 -> 3.5
   9a14f44bd..717c30891  3.4 -> 3.4
   9df9c61d6..d2b1be1b7  3.3 -> 3.3

**Comment by adamlazik1** [comment]
> Unfortunately, I've merged a "!fixup" commit into master. I was hoping GH would do a "real" rebase and automatically squash fixup the commit correctly. :shrug:

fixup commits only get merged to other commits if `--autosquash` is used. Though I forgot whether the rebase itself needs to be interactive or not but I know I have tested it before.

**Comment by maximiliankolb** [comment]
Yeah, I have it set up like this; but apparently it's not the default on Github UI.

**Pull request merged**